mint/setup_script.js

```diff
@@ -22,7 +22,7 @@ const LOCAL_KIOSK_PKG = "0xc25518ca0079c243b7973f5ebe3e8a390c71ab2f2a2cad1dcd11a
 const RANDOM_OBJ = "0x8";
 const CLOCK_OBJ = "0x6";
 // Environment-specific configurations
-const CONFIGS = {
+export const CONFIGS = {
```

Comment:
move  `CONFIGS` and the ADMIN key and `suiprivkeyToHex` to `config.ts`. Then you won't have issues with other functions , and you avoid unnecessary imports in this file and the hack with the "main"